static/translations/messages_en.json

```diff
@@ -242,5 +242,6 @@
   "Signed out": "Signed out",
   "Remove account?": "Remove account?",
   "This will make the mobile app on this device forget {email} on {realmUrl}.": "This will make the mobile app on this device forget {email} on {realmUrl}.",
-  "Remove account": "Remove account"
+  "Remove account": "Remove account",
+  "Failed to mark topic as read": "Failed to mark topic as read"
```

Comment:
nit: add a new string here on any line other than the last -- that way the commas don't cause the diff to unnecessarily touch another string :stuck_out_tongue: